pollux/vc-jwt/src/main/scala/org/hyperledger/identus/pollux/vc/jwt/JsonWebKey.scala

```diff
@@ -0,0 +1,72 @@
+package org.hyperledger.identus.pollux.vc.jwt
+
+import io.circe.*
+import io.circe.syntax.*
+
+case class JsonWebKey(
+    alg: Option[String] = Option.empty,
+    crv: Option[String] = Option.empty,
+    e: Option[String] = Option.empty,
+    d: Option[String] = Option.empty,
+    ext: Option[Boolean] = Option.empty,
+    key_ops: Vector[String] = Vector.empty,
+    kid: Option[String] = Option.empty,
+    kty: String,
+    n: Option[String] = Option.empty,
+    use: Option[String] = Option.empty,
+    x: Option[String] = Option.empty,
+    y: Option[String] = Option.empty
+)
+
+object JsonWebKey {
+  given jsonWebKeyEncoder: Encoder[JsonWebKey] =
+    (jsonWebKey: JsonWebKey) =>
+      Json
+        .obj(
+          ("alg", jsonWebKey.alg.asJson),
+          ("crv", jsonWebKey.crv.asJson),
+          ("e", jsonWebKey.e.asJson),
+          ("d", jsonWebKey.d.asJson),
+          ("ext", jsonWebKey.ext.asJson),
+          ("key_ops", jsonWebKey.key_ops.asJson),
+          ("kid", jsonWebKey.kid.asJson),
+          ("kty", jsonWebKey.kty.asJson),
+          ("n", jsonWebKey.n.asJson),
+          ("use", jsonWebKey.use.asJson),
+          ("x", jsonWebKey.x.asJson),
+          ("y", jsonWebKey.y.asJson),
+        )
```

Comment:
Any reason why we prefer manual encoder / decoder over derived one? https://circe.github.io/circe/codecs/semiauto-derivation.html